style/swift/README.md

```diff
@@ -5,6 +5,7 @@ Swift
 
 * Keep up with the Objective-C style guide above. Will highlight differences
   here.
+* Limit lines to 100 characters
```

Comment:
Curious why this is different that our general [rule](https://github.com/thoughtbot/guides/tree/master/style#formatting):

> Break long lines after 80 characters.